### ISSUE TITLE
GUI: vdisk management actions page shows wrong values for sco size and write buffer.

### ISSUE THREAD

**Issue by saelbrec** [openvstorage/framework#1408]
Following screenschots show the configured volume write buffer for vpool and the values on the vdisk detail management page for sco size and volume write buffer
![screen shot 2017-02-02 at 22 39 54](https://cloud.githubusercontent.com/assets/12733895/22570323/1f35d1c6-e99b-11e6-9c05-8f7eaaf6caf1.png)
![screen shot 2017-02-02 at 22 40 09](https://cloud.githubusercontent.com/assets/12733895/22570327/22074e16-e99b-11e6-955b-737e0a83b1bf.png)

But apparently the volume driver is correctly set to 512MB write buffer. We can tell from the following choking messages in its log file:
```
Feb 02 17:20:39 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:39 419591 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b475 - info - maybeChokeNamespaces_: Namespace 6bc7983c-70eb-4a37-8746-4726eb51700b non-disposable: 544 MiB, max: 512 - choking it
Feb 02 17:20:39 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:39 419657 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b476 - info - maybeChokeNamespaces_: Namespace 74ffe060-0553-4bc5-8263-1ebb2761669b non-disposable: 544 MiB, max: 512 - choking it
Feb 02 17:20:39 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:39 419693 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b477 - info - maybeChokeNamespaces_: Namespace 9eefec60-f3a7-41ad-9233-eadb0fa965fe non-disposable: 544 MiB, max: 512 - choking it
Feb 02 17:20:39 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:39 419771 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b478 - info - maybeChokeNamespaces_: Namespace f1baf163-9092-4756-8b1a-fc211ef34e6e non-disposable: 544 MiB, max: 512 - choking it
Feb 02 17:20:40 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:40 420002 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b487 - info - maybeChokeNamespaces_: Namespace 9373d560-e5c7-4518-8382-8f6ec484451e non-disposable: 544 MiB, max: 512 - choking it
Feb 02 17:20:40 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:40 420107 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b488 - info - maybeChokeNamespaces_: Namespace df93606a-da4a-47ec-a5dd-3a6a8f174b3c non-disposable: 544 MiB, max: 512 - choking it
Feb 02 17:20:41 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:41 420394 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b54c - info - maybeChokeNamespaces_: Namespace 6bc7983c-70eb-4a37-8746-4726eb51700b non-disposable: 416 MiB, max: 512 - unchoking it
Feb 02 17:20:41 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:41 420454 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b54d - info - maybeChokeNamespaces_: Namespace 74ffe060-0553-4bc5-8263-1ebb2761669b non-disposable: 480 MiB, max: 512 - unchoking it
Feb 02 17:20:41 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:41 420476 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b54e - info - maybeChokeNamespaces_: Namespace 9373d560-e5c7-4518-8382-8f6ec484451e non-disposable: 480 MiB, max: 512 - unchoking it
Feb 02 17:20:41 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:41 420493 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b54f - info - maybeChokeNamespaces_: Namespace 9eefec60-f3a7-41ad-9233-eadb0fa965fe non-disposable: 480 MiB, max: 512 - unchoking it
Feb 02 17:20:41 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:41 420521 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b550 - info - maybeChokeNamespaces_: Namespace df93606a-da4a-47ec-a5dd-3a6a8f174b3c non-disposable: 480 MiB, max: 512 - unchoking it
Feb 02 17:20:41 ftcmp0 volumedriver_fs.sh[31010]: 2017-02-02 17:20:41 420564 +0100 - ftcmp0 - 31010/0x00007f08b8736700 - volumedriverfs/SCOCache - 000000000000b551 - info - maybeChokeNamespaces_: Namespace f1baf163-9092-4756-8b1a-fc211ef34e6e non-disposable: 480 MiB, max: 512 - unchoking it
```

So I assume there is an issue somewhere in the GUI or FWK which does not correctly take the configure block size of 8k(but rather 4k) into account for its calculation of the wrong displayed values.

**Comment by khenderick** [comment]
Fixed by #1411, packaged on dev openvstorage-2.7.12-dev.1486374024.de8e760

**Comment by khenderick** [comment]
openvstorage-2.7.12.1
